@here/harp-geoutils/lib/tiling/TileTreeTraverse.ts

```diff
@@ -16,18 +15,22 @@ export class TileTreeTraverse {
     }
 
     subTiles(tileKey: TileKey): TileKey[] {
-        const subTileCount =
-            this.m_subdivisionScheme.getSubdivisionX(tileKey.level) *
-            this.m_subdivisionScheme.getSubdivisionY(tileKey.level);
+        const level = tileKey.level;
+        const divX = this.m_subdivisionScheme.getSubdivisionX(level);
+        const divY = this.m_subdivisionScheme.getSubdivisionY(level);
 
-        // tslint:disable-next-line:no-bitwise
-        const subTileMask = ~(~0 << subTileCount);
-
-        const subTiles = new SubTiles(tileKey, 1, subTileMask);
```

Comment:
I guess SubTiles was added for testability, but I think we could also add a test that checks TileTreeTraverse.subTiles directly. So for me it's fine to remove SubTiles but maybe also double check with Andres(Mandado) b/c he created that class.

Reply:
Fortunately, I've added test cases for TileTreeTraverse that fails with current SubTiles implementation.
https://github.com/heremaps/harp.gl/pull/683/files